dev/buildtool/util.py

```diff
@@ -81,17 +82,35 @@ def run_subprocess(cmd, stream=None, echo=False, **kwargs):
       stdout=subprocess.PIPE,
       stderr=subprocess.STDOUT,
       **kwargs)
+  logging.log(log_level, 'Running %s as pid %s', split_cmd[0], process.pid)
+  process.start_date = start_date
+
   time.sleep(0) # yield this thread
+  return process
```

Comment:
Is the `time.sleep(0)` still needed here? I thought this didn't produce the yield as we expected.

Reply:
It wont hurt. It didnt go to the extent I thought, but might still do something, especially in other use cases that arent under as much load.